### DISCUSSION TITLE
Consolidating Google Sheet Rows Between Consumer Sheets

### DISCUSSION THREAD

**Discussion by Flywheeltargeting** [merchantprotocol/theadslab#22] [Open Discussion Forum]
Google sheets rows between sheets. 

What is the best way to match up the visitors ID across google sheet pages. I.E. Row 2 on consumer  PII, consumer phone and consumer social email dont match up with the same user.

**Comment by jonathonbyrdziak** [comment]
The **ID** is the linking match between all your data tables, you can also use the **HOUEHOLD_ID** or **ADDRESS_ID** respectively
